_docs/tasks/configuring-mixer.md

```diff
@@ -0,0 +1,163 @@
+---
```

Comment:
I feel throughout you're putting ticks around things that don't need it. Ticks is mostly for code items and config elements. See the style guide for more specifics.